src/python/py_generate_pipeline.cpp

```diff
@@ -0,0 +1,153 @@
+    }
+}
+
+void update_config_from_kwargs(ov::GenerationConfig& config, const py::kwargs& kwargs) {
+    if (kwargs.contains("max_new_tokens")) config.max_new_tokens = kwargs["max_new_tokens"].cast<size_t>();
+    if (kwargs.contains("max_length")) config.max_length = kwargs["max_length"].cast<size_t>();
+    if (kwargs.contains("ignore_eos")) config.ignore_eos = kwargs["ignore_eos"].cast<bool>();
+    if (kwargs.contains("num_beam_groups")) config.num_beam_groups = kwargs["num_beam_groups"].cast<size_t>();
+    if (kwargs.contains("num_beams")) config.num_beams = kwargs["num_beams"].cast<size_t>();
+    if (kwargs.contains("diversity_penalty")) config.diversity_penalty = kwargs["diversity_penalty"].cast<float>();
+    if (kwargs.contains("length_penalty")) config.length_penalty = kwargs["length_penalty"].cast<float>();
+    if (kwargs.contains("num_return_sequences")) config.num_return_sequences = kwargs["num_return_sequences"].cast<size_t>();
+    if (kwargs.contains("no_repeat_ngram_size")) config.no_repeat_ngram_size = kwargs["no_repeat_ngram_size"].cast<size_t>();
+    if (kwargs.contains("stop_criteria")) str_to_stop_criteria(config, kwargs["stop_criteria"].cast<std::string>());
+    if (kwargs.contains("temperature")) config.temperature = kwargs["temperature"].cast<float>();
+    if (kwargs.contains("top_p")) config.top_p = kwargs["top_p"].cast<float>();
+    if (kwargs.contains("top_k")) config.top_k = kwargs["top_k"].cast<size_t>();
+    if (kwargs.contains("do_sample")) config.do_sample = kwargs["do_sample"].cast<bool>();
+    if (kwargs.contains("repetition_penalty")) config.repetition_penalty = kwargs["repetition_penalty"].cast<float>();
+    if (kwargs.contains("pad_token_id")) config.pad_token_id = kwargs["pad_token_id"].cast<int64_t>();
+    if (kwargs.contains("bos_token_id")) config.bos_token_id = kwargs["bos_token_id"].cast<int64_t>();
+    if (kwargs.contains("eos_token_id")) config.eos_token_id = kwargs["eos_token_id"].cast<int64_t>();
+    if (kwargs.contains("eos_token")) config.eos_token = kwargs["eos_token"].cast<std::string>();
+    if (kwargs.contains("bos_token")) config.bos_token = kwargs["bos_token"].cast<std::string>();
+}
+
+// operator() and generate methods are identical, operator() is just an alias for generate
+std::string call_with_kwargs(ov::LLMPipeline& pipeline, const std::string& text, const py::kwargs& kwargs) {
+    // Create a new GenerationConfig instance and initialize from kwargs
+    ov::GenerationConfig config = pipeline.get_generation_config();
+    update_config_from_kwargs(config, kwargs);
+    return pipeline(text, config);
+}
+
+std::string call_with_config(ov::LLMPipeline& pipe, const std::string& text, const ov::GenerationConfig& config) {
+    std::shared_ptr<StreamerBase> streamer;
+    return pipe(text, config);
+}
+
+std::string ov_tokenizers_module_path() {
+    py::module_ m = py::module_::import("openvino_tokenizers");
```

Comment:
with the latest version it will look by default `openvino_tokenizers.dll/so` is located in the same folder and only if it's not there will try to call `import openvino_tokenizer`

```
std::string ov_tokenizers_module_path() {
    // Try a path relative to build artifacts folder first.
    std::filesystem::path from_library = with_openvino_tokenizers(get_ov_genai_bindings_path());
    if (std::filesystem::exists(from_library)) {
        return from_library.string();
    }
    return py::str(py::module_::import("openvino_tokenizers").attr("_ext_path"));
}
```